docs/examples/scheduled-capacity.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       # Scale way up for higher traffic for weekdays during work hours
       - replicas: 6
         start:
-          weekdays: mon,tue,wed,thu,fri
+          weekdays: 1,2,3,4,5
```

Comment:
This makes me really confident that we shouldn't support numbered weekdays. What on earth is 1. Is it zero indexed? Very unintuitive.

Reply:
I'm basing it off of how crontab standard. You're right, it's not really clear if it's 0- or 1-indexed, I like having it though, for people who do understand the tooling better or for those who already have an existing crontab setup to transfer it easily.

Reply:
You own this, so I'm not going to hard block you here, but this is a great example of voluntary complexity and sprawling API surface. It's is an explicit anti-pattern and generates pain for both developers and customers.